docs/tutorial.md

```diff
@@ -39,7 +39,7 @@ This tutorial will walk through creating a kpack [builder](builder.md) resource
    
    > Note: Learn more about kpack secrets with the [kpack secret documentation](secrets.md) 
 
-1. Create a service account that references the registry secret created above 
+2. Create a service account that references the registry secret created above
```

Comment:
Is there a reason to move to numbering explicitly? Mardown will number correctly with 1.s

Reply:
my preference is usually just to explicitly number them to keep it simple